### PR TITLE
Creating EIP-1202 DRAFT

### PR DESCRIPTION
**Pull request by xinbenlv**
Please discuss this PR and EIP at #1202

### CODE REVIEW

**Comment by xinbenlv** [comment]
Ready to check-in.

**Comment by xinbenlv** [comment]
Dear editor team, how do I request the PR to be merged?

**Comment by xinbenlv** [comment]
Bump~

File: EIPS/eip-1202.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+// TODO: provide example interface and implementation
+
+### Example 2: Most Sophisticated Version: Multi-Issue Multi-Option Ordered-Result Override-able Privacy-Safe ERC-20/or other interface based Weighted Vote
+// TODO: provide example interface and implementation
+
+### Example 3: Somewhere-in-between Example 1 and Example 2 (To-Be-Discussed)
+// TODO: provide example interface and implementation
+
+## Specificiations
+// TODO
```
Comment (Arachnid) [inline EIPS/eip-1202.md]: Please actually have a standard before you submit a PR to add a draft to the repository.

Comment (xinbenlv) [reply EIPS/eip-1202.md]: Thanks. I created #1237 and please take look if that is what you refer to?

**Pull request closed without merging**